jenkins_job_wrecker/modules/publishers.py

```diff
@@ -132,14 +132,21 @@ def buildtrigger(top, parent):
             build_triggers = []
             for sub in element:
                 project = {}
+                node_parameter_exist = False
+                current_parameters_exist = False
                 for config in sub:
                     if config.tag == 'projects':
-                        project['project'] = config.text
+                        if len(config.text.split(',')) > 1:
+                            project['project'] = [x.strip() for x in config.text.split(',')]
+                        else:
+                            project['project'] = config.text
```

Comment:
JJB API says it should be a list so maybe this else condition is not needed?

Reply:
I was trying to replicate JJB but you are right.